src/core/device.cpp

```diff
@@ -256,6 +256,14 @@ namespace occa {
     return modeDevice->createStream(streamProperties(props));
   }
 
+  stream device::wrapStream(void* ptr, const occa::json &props) {
+    assertInitialized();
+
```

Comment:
Actually, passing a nullptr to the CPU backends should be acceptable, since I'm not sure what the use would even pass for that case. I've added this error check in the respective backends instead.